.github/workflows/lint-pytest.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.10", "3.11"]
```

Comment:
didn't we decide to move completely to 3.11?

Reply:
ok, moving it back in

Reply:
although what is the benefit of having both? 3.11 is already long enough around, and anyway the users are expected to install a seperate environemnt. What are the benefits to also having 3.10 around @maurerle?

Reply:
I think there are still some installs which default to 3.10 (the conda base env in my conda 23.5.2).
People will still be using python 3.10, want to use assume in their existing env or do something else.

If we drop 3.10 from testing, we might not see, when we break compatibility (or one of our dependencies breaks 3.10 compatibliity).
If this is the case, we can then decide if we want to drop 3.10.

In that case we should also increase `python = "^3.10"` to `python = "^3.11"` in the pyproject.toml, so that assume can not be installed on python < 3.10

Reply:
ok